snabbdom/src/test/scala/snabbdom/SnabbdomSuite.scala

```diff
@@ -1357,7 +1357,7 @@ class SnabbdomSuite extends BaseSuite {
       patch(vnode0, vnode1)
       patch(vnode1, vnode2)
       assertEquals(result1.length, 1)
-      assertEquals(result2.length, 1)
+      assertEquals(result2.length, 0)
```

Comment:
And this is the optimization kicking-in, I believe. It only affects the text node, since the text is unchanged in this instance.